<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1253</width>
    <height>1014</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>355</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Operating Scenario</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>561</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Components Status</string>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>541</width>
       <height>171</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>561</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Scenario Control</string>
    </property>
    <widget class="QPushButton" name="btn_scenario_run">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_scenario_stop">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_scenario_pause">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_scenario_resume">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>Resume</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_scenario_stepover">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>Step Over</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="table_scenario_contents">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>661</width>
      <height>921</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1253</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Open operating scenario</string>
   </property>
   <property name="shortcut">
    <string>Meta+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Program Exit</string>
   </property>
   <property name="shortcut">
    <string>Meta+X</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
